app/src/main/java/com/connorlay/knitgrid/fragments/MulticolorFragment.java

```diff
@@ -0,0 +1,91 @@
+package com.connorlay.knitgrid.fragments;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.DialogInterface.OnClickListener;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+import android.view.LayoutInflater;
+import android.view.View;
+
+import com.connorlay.knitgrid.R;
+import com.connorlay.knitgrid.presenters.PatternPresenter;
+
+/**
+ * Created by Greg on 4/5/2016.
+ */
+public class MulticolorFragment extends DialogFragment {
```

Comment:
This should be renamed MulticolorDialogFragment, to make it easy to tell it's just a dialog.